client/src/utils/data-formats/formats/timeline.ts

```diff
@@ -64,18 +64,21 @@ export function timeline(
   let values = state[args.data || 'values'];
   const timeline = values;
 
-  let _timeline = {};
+  let _timeline: { [time: number] : object} = {};
```

Comment:
P.s. regarding you aversion to "time" sort of writing, you could expose a `export const TIME = "time";` from `timeline.ts` where Timeline.tsx could consume, this way, you online define "time" once.